src/test/java/api/requests/RequestsAPICreationTests.java

```diff
@@ -4979,6 +4979,51 @@ void printAndFetchRequestWithPrintEventFeatureDisabled() {
     assertThat("printDetails should be null for request2 because the print event feature is disabled",
       requestRepresentation2.getJsonObject("printDetails"), Matchers.nullValue());
   }
+
+  @Test
+  void itemLevelRequestCreationShouldFailDueToPickUpServicePointNotAllowed() {
```

Comment:
Is that what really happens in this test?

Reply:
sorry, forgot to rename it